bittensor/utils/networking.py

```diff
@@ -178,22 +163,49 @@ def get_formatted_ws_endpoint_url(endpoint_url: Optional[str]) -> Optional[str]:
 def ensure_connected(func):
     """Decorator ensuring the function executes with an active substrate connection."""
 
+    def is_connected(substrate) -> bool:
+        """Check if the substrate connection is active."""
+        sock = substrate.websocket.sock
+        return (
+            sock is not None
+            and sock.getsockopt(socket.SOL_SOCKET, socket.SO_ERROR) == 0
+        )
+
+    @retry(
+        exceptions=ConnectionRefusedError,
+        tries=5,
+        delay=5,
+        backoff=1,
```

Comment:
What do you think of adding a backoff of 1.5 instead? 
So instead of retries every 5 seconds, the time it waits will increase: `5, 7.5, 11,25, 16...`

I was thinking it might be good if the subtensor connection takes a bit more time to reinitialize

Reply:
My thinking revolved around the local nodes which validators would usually run and if they are down, then it might take a while to start. 

However, after our discussion today in the standup regarding websocket subscriptions etc, I think standard increments are okay. The issue is something else.